Documentation/git-status.txt

```diff
@@ -207,6 +209,76 @@ field from the first filename).  Third, filenames containing special
 characters are not specially formatted; no quoting or
 backslash-escaping is performed.
 
+Very Verbose Porcelain Format
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When --verbose is given twice along with --porcelain, additional output
+is provided.
+
+If --branch is given, the first line shows a summary of the current
+operation in progress.  This line begins with "### state: ", the name
+of the operation in progress, and then operation-specific information.
+Fields are separated by a single space, unless otherwise indicated.
+
+    Operation   Fields                     Explanation
+    --------------------------------------------------------
+    clean
+    merge       <nr>                       Number unmerged
+    am          ["E"]                      Present if empty path
+    rebase      <nr>                       Number unmerged
+                ["S"]                      Present if split
+                ["E"]                      Present if edit
```

Comment:
What does "split" or "edit" mean in this context? If there are unmerged entries, is the "S" or the "E" appended?